src/models/ServerStyleSheet.js

```diff
@@ -48,6 +48,18 @@ class ServerTag implements Tag {
     return `<style type="text/css" ${namesAttr} ${localAttr}>\n${css}\n</style>`
   }
 
+  toElement() {
+    const attributes = {
+      [SC_ATTR]: this.names.join(' '),
+      [LOCAL_ATTR]: this.isLocal,
```

Comment:
The result written to file is something like this:

```html
<style type="text/css" data-styled-components="biyTF hdZKtG gqDZGS clHZEQ yOoIr fUPLnt inUVIL fSmkBB kZQtlQ PIsWY fPJRNv jgRNmm TsYdo iZXahJ bDCuCY kUYNDb foZtll cqjtay fjMziR gqhtmN cEnRUY dhfSfb NkGiE hFJQDZ cOZxoD jYtjIs jskQpj fIEXtD isATaW btuKsl dReRJd dhLCfp gESlQT hoNHWU hRbfbz bMefOW cHPONr fFcRsM" data-styled-components-is-local="true">
```

You mean it wouldn't work if React omits `="true"`?